### ISSUE TITLE
context menus should have menu semantics

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#848]
Currently our context menus don't have the same screenreader experience as, for example, the chrome context menu or the chrome settings menu. That is, they don't announce 'menu submenu', nor 'one of eighteen'.   We should ensure that the semantics of the context menus are standardized according to at least user expectations.
